### ISSUE TITLE
Kaspersky_assets does not exist

### ISSUE THREAD

**Issue by diogofgm** [diogofgm/kaspersky_app#4]
Hi,
after clic on Help in Kaspersky App, I have NOK above Assets Lookup File with a similar message "The lookup table 'kaspersky_assets.csv' is invalid."

splunk btool props list --debug | grep -i kaspersky_assets  show nothing

__Originally posted by @Staj-s in https://github.com/diogofgm/kaspersky_app/issues/3#issuecomment-712215245__

**Comment by diogofgm** [comment]
@Staj-s there is a saved search that creates that file for you. The dashboard uses the file it self that's why you seen no actual lookup definition in conf. Also you should be looking in transforms not props with btool since, unless it's and automatic lookup, you won't have a props entry. 
Nevertheless I'll add the configuration for that in a fix release.

**Comment by Staj-s** [comment]
Hi, 
splunk btool transforms  list --debug | grep -i kaspersky_assets  show nothing.
I don't find the reason for my issue.

don't hesitate to tell me when you release a fix release
Thanks for your response.

**Comment by diogofgm** [comment]
Check in /opt/splunk/etc/apps/kaspersky_app/lookups/
The file in question should be there. If it not, check if the saved search that ships with the app is enabled and you can even run it since it creates the file. 
Regarding the btool it's ok in this particular case that you don't get anything since the file itself it's not setup as a lookup. I was just giving you some pointers that might be true in other situations.

**Comment by Staj-s** [comment]
Indeed, i have a empty file kaspersky_assets.csv in date of 22 oct, i.e date of installation of TA-kaspersky

**Comment by Staj-s** [comment]
must i remove this kaspersky_assets.csv file ?

**Comment by diogofgm** [comment]
No. Keep the file there. The only reasons that file is empty it's either the saved search is not running or the macro with the index name still has the default index name and not the one you're using for you data.

Have you changed the macro kaspersky_indexes? By default it's index=epav
If this isn't the index where you're storing you kaspersky data change it to the proper index.
After changing the macro run the saved search that ships with the app to fill that file.

**Comment by Staj-s** [comment]
yes i changed to index=epav

**Comment by diogofgm** [comment]
Ok if you have your kaspersky data in the epav index try to run the saved search and check if it gets results.

The search is called kaspersky_asset_lookup_gen
